tools/get_release_contributors.py

```diff
@@ -127,12 +134,12 @@ def get_user_by_email(self, key):
             return None
         email = m.group(1)
         # Handle Github-generated email addresses via static matching:
-        m = re.match(r'\A(\d+\+)?([^+@]+)\@users\.noreply\.github\.com\Z', email)
+        m = re.match(r'\A(\d+\+)?([^+@]+)@users\.noreply\.github\.com\Z', email)
         if m:
             return m.group(2)
         r = self._github_api_get('search/users', params={'q': f'{email} in:email'})
         if r.status_code < 200 or r.status_code >= 300:
-            logger.warning(f'search/users for {email} failed with code {r.status_code}')
+            logger.warning('search/users for %s failed with code %s', email, r.status_code)
```

Comment:
Same question as above: Why non format string here.

Reply:
If you want I can paste the longer errors and warnings from the current default branch HEAD. It explains a few things with links. 

In short,  logging in Python should be done with (lazy) printf-style templates

Reply:
Ok. If the standard for logging is different to other forms of output, that's ok.

Reply:
Yes - the linter has a rule for this:
```
tools/get_release_contributors.py:144:8: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)
```

Reply:
Ok. Thanks